web/assets/main.scss

```diff
@@ -77,3 +77,7 @@ $govuk-assets-path: "/lpa/frontend/assets/";
         border-bottom: none;
     }
 }
+
+.app-\!-select-draft-width {
+  width: 88%;
+}
```

Comment:
Not ideal but the grid classes available to set the width don't go wide enough (the widest class `govuk-grid-column-three-quarters` only sets the width to 75%).
<img width="799" alt="Screenshot 2023-01-20 at 15 49 41" src="https://user-images.githubusercontent.com/51047911/213744857-6f52ea10-2b08-418b-b10f-06d338e04c33.png">

Reply:
Could you just go full width with `govuk-grid-column-full`?

Reply:
To get back to your comment below @gregtyler , `govuk-grid-column-full` doesn't work as i don't want the drop down to span the full width as it knocks the select button to sit below it. The next widest grid class `govuk-grid-column-three-quarters` is not wide enough.

Reply:
Ah sorry, I misunderstood what this was applying to. In that case it'd probably be safer to use flexbox, but this setup will work fine.

Reply:
Switched to flexbox, as it was cleaner